app/models/miq_ae_namespace.rb

```diff
@@ -82,7 +99,7 @@ def self.find_tree(find_options = {})
   end
 
   def fqname
-    @fqname ||= "/#{ancestors.collect(&:name).reverse.push(name).join('/')}"
+    @fqname ||= "/#{path.pluck(:name).join('/')}"
```

Comment:
can you store `fqname` in the db?
(as long as you don't store domain, this could be very useful)

then just query that

Reply:
response: saving this for pr2. going to probably split these into 2 columns and all that logic will be better suited for phase 2